dcos-log/api/v2/handlers.go

```diff
@@ -605,5 +602,8 @@ func downloadFile(w http.ResponseWriter, req *http.Request) {
 		}
 	}
 
-	io.Copy(w, downloadResp.Body)
+	_, err = io.Copy(w, downloadResp.Body)
+	if err != nil {
+		logError(w, req, err.Error(), http.StatusInternalServerError)
```

Comment:
if we couldn't write the resp to `w` does it make sense to logError here?  maybe it makes sense to just log the error without trying to write to `w`?